node/service/Cargo.toml

```diff
@@ -5,59 +5,72 @@ authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2018"
```

Comment:
GitHub doesn't let me comment on the proper line, but maybe this crate should be renamed to `polkadot-service`?

Reply:
838248b